tox.ini

```diff
@@ -58,7 +58,6 @@ commands =
     lint: pylint --rcfile=tests/pyproject.toml tests
     lint: pydocstyle report tests bin
     lint: pycodestyle report tests bin
-    dockercompose: docker-compose {posargs}
```

Comment:
Part of the `docker-compose` &rarr; `docker compose` change that was made in the cookiecutter